### PR TITLE
Enabling use of VS Code for debugging

### PR DESCRIPTION
**Pull request by mechaffin**
# Description and Motivation 

Enabling use of VS Code for debugging. Improve experience/documentation.

# Change type 

- [x] Bug fix
- [ ] New feature
- [ ] Enhancement
- [ ] Breaking change (breaks backward compatibility)

**Checklist:**

- [x] All tests passed
- [x] The code follows the code style and conventions of this project
- [x] The change requires a change to the documentation
- [x] I have updated the documentation accordingly

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "env": {
+                // Note: Any setting made in this file will override the operating system environment variable of that name.
+                // See https://docs.microsoft.com/aspnet/core/fundamentals/environments
+                "ASPNETCORE_ENVIRONMENT": "Development",
+                // Environment variables used in appsettings.ini.  
+                // Required Settings. 
+                "PCS_TELEMETRY_DOCUMENTDB_CONNSTRING": "your DocumentDB connection string",
```
Comment (zhang-hua) [inline .vscode/launch.json]: nit: use {...} might be helpful to indicate the value need to be replaced.

**Comment by zhang-hua** [comment]
After we merge this PR, we might still need to add .vscode in .gitignore to avoid commit this file with credentials by mistake.

File: README.md [hunk 2]
```diff
@@ -55,20 +55,21 @@ for more information. More information on environment variables
   * `PCS_STORAGEADAPTER_WEBSERVICE_URL` = http://localhost:9022/v1
 
 ## Running the service with Visual Studio
```
Comment (ppathan) [inline README.md]: In other PRs and template you include Vistual Studio and VS Code. We should match.

File: README.md [hunk 3]
```diff
@@ -109,31 +110,39 @@ required to package the service into a Docker image:
...
+import credentials and network details. Environment variables are not
+mandatory though, you can for example edit appsettings.ini and write
+credentials directly in the file. Just be careful not sharing the changes,
+e.g. sending a Pull Request or checking in the changes in git.
+
+The configuration file in the repository references some environment
+variables that need to be defined. Depending on the OS and the IDE used,
+there are several ways to manage environment variables.
+
+1. If you're using Visual Studio or Visual Studio for Mac, the environment
```
Comment (ppathan) [inline README.md]: nit: To use Visual Studio for Windows or Mac

File: README.md (hunk 2 above)
Comment (molly-moen) [reply README.md]: I would prefer splitting visual studio and vs code instructions into two sections, or at least make clear which pre-reqs are needed for which ide

**Pull request merged**